src/pages/docs/platform/architecture/performance.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Performance
```

Comment:
This page needs latency data or to refer to latency data to evidence Ably's ability to provide the low latencies we refer to. There's no indication currently what is considered low latency, we should add that now and link to our current [global latency page ](https://ably.com/global-round-trip-latencies) and robust latency data when we have it.

Reply:
should we do that as a later change?

Reply:
> should we do that as a later change?

Adding the detailed latency data, yes. For now though we could at least refer to our sub 50ms (not sure what number we're using atm) global latency and link out to the global latency page (uptrends data).